test/distributed/tensor/parallel/test_tp_style.py

```diff
@@ -67,6 +67,27 @@ def test_colwise_parallel_style(self):
             self.assertEqual(comm_mode.get_comm_counts()[c10d_functional.reduce_scatter_tensor], 1)
             self.assertEqual(comm_mode.get_total_counts(), 2)
 
+    @with_comms
+    def test_colwise_parallel_embedding(self):
```

Comment:
so colwise embedding sharding is supported. How is colwise embedding sharding max norm different?

Reply:
max norm embedding is not working at all, feel free to explore and see what's the gap there.